### ISSUE TITLE
Wrong error message for `msg.sender.call{value: amount}("");`

### ISSUE THREAD

**Issue by avatar-lavventura** [duaraghav8/Ethlint#286]
For the following line: `(bool success, ) = msg.sender.call{value: amount}("");` ; I am getting following error message:
```
Syntax error: unexpected token {
✖ 1 error found.
```

--------

But `solidity 0.7.0` recommend us to use `{value: ...}` an usage of `{` seems correct. Why solium generates this error message even it is correct and recommended usage in Solidity? 

```
TypeError: Using ".value(...)" is deprecated. Use "{value: ...}" instead.
        (bool success, ) = msg.sender.call.value(amount)(""); 
```
